src/libs/manymouse/linux_evdev.c

```diff
@@ -48,7 +50,7 @@ static int poll_mouse(MouseStruct *mouse, ManyMouseEvent *outevent)
     while (unhandled)  /* read until failure or valid event. */
     {
         struct input_event event;
-        int br = read(mouse->fd, &event, sizeof (event));
+        const ssize_t br = read(mouse->fd, &event, sizeof (event));
```

Comment:
For every file you have changed, please mention it at the header (with your real name or GitHub nick) - as the Zlib license requires us to clearly mark that this is not an original code.

Reply:
Thanks! will do :+1:

Reply:
Added.